client/login/magic-login/index.jsx

```diff
@@ -278,10 +278,8 @@ class MagicLogin extends Component {
 		const { isJetpackLogin, locale, showCheckYourEmail, translate, isWCCOM, query } = this.props;
 
 		const isA4A = query?.redirect_to?.includes( 'agencies.automattic.com/client' ) ?? false;
-		const { isiPad, isiPod, isiPhone, isAndroid } = userAgent;
-		const isMobile = isiPad || isiPod || isiPhone || isAndroid;
 
-		const hideAppPromo = isA4A || ! isMobile;
+		const hideAppPromo = isA4A || ! isMobile();
```

Comment:
Here `! ( isIos() || isAndroid() )` makes more sense because the app promo promotes the Jetpack Mobile App which is not available on any other mobile OS.

Reply:
> Here `! ( isIos() || isAndroid() )` makes more sense because the app promo promotes the Jetpack Mobile App which is not available on any other mobile OS.

Updated in 32b12dd8b07.